aries_vcx/src/common/ledger/transactions.rs

```diff
@@ -115,7 +116,7 @@ pub async fn into_did_doc(profile: &Arc<dyn Profile>, invitation: &Invitation) -
                     error!("Failed to obtain service definition from the ledger: {}", err);
                     AriesService::default()
                 });
-            (service.service_endpoint, service.recipient_keys, service.routing_keys)
+            (service.service_endpoint, did_key_to_public_key(service.recipient_keys), service.routing_keys)
```

Comment:
@Patrik-Stas  is the intention of #689 to just support did:key conversion for OOB recipient_keys? there is also `routing_keys`, and also a pairwise invitation which may or may not be in `did:key` format. There are probably several other spots where `did:key`s can be input